fdbclient/MonitorLeader.actor.cpp

```diff
@@ -968,11 +989,16 @@ ACTOR Future<MonitorLeaderInfo> monitorProxiesOneGeneration(
 			shrinkProxyList(ni, lastCommitProxyUIDs, lastCommitProxies, lastGrvProxyUIDs, lastGrvProxies);
 			clientInfo->setUnconditional(ni);
 			successIndex = index;
+			allConnectionsFailed = false;
 		} else {
 			TEST(rep.getError().code() == error_code_failed_to_progress); // Coordinator cant talk to cluster controller
```

Comment:
As you are adding the traceevent to log the error, we can remove the 2 TEST() lines.

Reply:
Indeed, the TEST() is for test coverage only